README.md

```diff
@@ -1,4 +1,7 @@
-Scanamo [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.gu/scanamo_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.gu/scanamo_2.12) [![Build Status](https://travis-ci.org/scanamo/scanamo.svg?branch=master)](https://travis-ci.org/scanamo/scanamo) [![Coverage Status](https://coveralls.io/repos/github/guardian/scanamo/badge.svg?branch=master)](https://coveralls.io/github/guardian/scanamo?branch=master) [![Chat on gitter](https://badges.gitter.im/guardian/scanamo.svg)](https://gitter.im/guardian/scanamo)
+| CI | Coverage | Release | Issues | Users | Chat |
+| --- | --- | --- | --- | --- | --- |
+| [![Build Status][Badge-Travis]][Link-Travis] | [![Coverage Status][Badge-Codecov]][Link-Codecov] | [![Release Artifacts][Badge-MavenReleases]][Link-MavenReleases] | [![Average time to resolve an issue][Badge-IsItMaintained]][Link-IsItMaintained] | [![Scaladex dependencies badge][Badge-Scaladex]][Link-Scaladex] | [![Gitter][Badge-Gitter]][Link-Gitter] |
+
 =======
```

Comment:
I think this double line is useless
<img width="987" alt="screen shot 2018-12-10 at 09 38 09" src="https://user-images.githubusercontent.com/233326/49723687-569ca800-fc5f-11e8-9a7a-04f97fa0801b.png">